config/acme/machines/config_batch.xml

```diff
@@ -100,13 +101,14 @@
       <arg flag="-q" name="$JOB_QUEUE"/>
       <arg flag="-l walltime=" name="$JOB_WALLCLOCK_TIME"/>
       <arg flag="-A" name="$PROJECT"/>
+      <arg flag="-M" name="$MAIL_USER"/>
+      <arg flag="-m" name="$MAIL_TYPE"/>
```

Comment:
Remove these. Instead add a <batch_email_type_flag> attribute to the batch system.

Example:
<batch_email_type_flag>, aeb, b, e, a</batch_email_type_flag>